test/evp_extra_test.c

```diff
@@ -873,6 +881,7 @@ static int test_set_get_raw_keys_int(int tst, int pub)
     }
 
     if (!TEST_ptr(pkey)
+            || !TEST_int_eq(EVP_PKEY_cmp(pkey, pkey), 1)
```

Comment:
This should use EVP_PKEY_eq()..

Reply:
This is on 1.1.1 branch, no EVP_PKEY_eq there.